src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/OpenTelemetryProtocolExporterEventSource.cs

```diff
@@ -94,6 +103,12 @@ public void TrySubmitRequestException(string ex)
         this.WriteEvent(12, ex);
     }
 
+    [Event(13, Message = "Error while attempting to re-transmit data from disk. Message: '{0}'", Level = EventLevel.Error)]
```

Comment:
Not for this PR but it would be nice to also log the telemetry signal that was being sent. We should do it for other methods in this class too.